contracts/test-utils/CHANGELOG.json

```
@@ -1,4 +1,18 @@
 [
+    {
+        "version": "3.1.0",
+        "changes": [
+            {
+                "note": "Create RichRevertReason classes"
+            },
+            {
+                "note": "Update assertions to support RichRevertReason objects"
```

```suggestion
                "note": "Update `assertions` to support `RichRevertReason` objects"
```
Like that. Just a nit